typescript/examples/vercel-ai/polymarket/index.ts

```diff
@@ -20,7 +21,14 @@ const walletClient = createWalletClient({
     chain: polygon,
 });
 
-(async () => {
+const terminal = readline.createInterface({
```

Comment:
There's 3 changes in this file:
1. i added a interactive terminal 
2. i added streaming responses 
3. i catch errors and feed them back to the LLM so it can continue